src/Terminus.php

```diff
@@ -2,47 +2,90 @@
 
 namespace Pantheon\Terminus;
 
-use Symfony\Component\Console\Application;
-use Symfony\Component\Console\Input\InputOption;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+use Robo\Runner;
 
-class Terminus extends Application
+class Terminus
 {
+    /**
+     * @var \Robo\Runner
+     */
+    private $runner;
 
     /**
-     * @inheritdoc
+     * Object constructor
      */
     public function __construct()
     {
+        $commands_directory = __DIR__ . '/Commands';
+        $top_namespace = 'Pantheon\Terminus\Commands';
+        $commands = $this->discoverCommands(['path' => $commands_directory, 'namespace' => $top_namespace,]);
+        $this->setRunner(new Runner($commands));
+    }
+
+    /**
+     * Runs the instantiated Terminus application
+     *
+     * @param string[] $arguments Argv from the command line
+     * @return integer The exiting status code of the application
+     */
+    public function run(array $arguments = [])
+    {
         $config = new Config();
-        parent::__construct('Terminus', $config->get('version'));
+        $status_code = $this->runner->execute($arguments, null, 'Terminus', $config->get('version'));
```

Comment:
You should construct your own output object here, and pass it to Robo. Then, you'll have a reference to your output object, and can use it to construct your logger.